Saule/Http/JsonApiConfiguration.cs

```diff
@@ -22,6 +22,11 @@ public class JsonApiConfiguration
         /// <summary>
         /// Gets the expressions that are used to evaluate filter queries on a per-type basis.
         /// </summary>
-        public QueryFilterExpressionCollection QueryFilterExpressions { get; } = new QueryFilterExpressionCollection();
+        public QueryFilterExpressionCollection QueryFilterExpressions { get; } = new QueryFilterExpressionCollection();
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the new experimental graph serializer is used.
+        /// </summary>
+        public bool UseGraphSerializer { get; set; } = false;
```

Comment:
Sure, I've moved the new to be in place of the old and removed the config option.